cmd/helm/install.go

```diff
@@ -462,3 +464,25 @@ func checkDependencies(ch *chart.Chart, reqs *chartutil.Requirements) error {
 	}
 	return nil
 }
+
+//readFile load a file from the local directory or a remote file with a url.
+func readFile(filePath string) ([]byte, error) {
+	_, err := os.Stat(filePath)
+	if err != nil {
+
+		resp, err := http.Get(filePath)
+		if err != nil {
+			return nil, err
```

Comment:
This will cause non-existent files to return error `Get /path/to/filename: unsupported protocol scheme ""` instead of `open /path/to/filename: no such file or directory`, which IMO is a regression in user experience.

I think you need to differentiate between filename and url, the simplest way you could implement it would be by using `net/url.Parse`. You can check if the string is http url if it parses and has http/https scheme.